### PR TITLE
Deployment

### PR DESCRIPTION
**Pull request by softbobo**
preparations for the deployment, as listed in #10

### CODE REVIEW

**Comment by obitech** [comment]
Before merging this, we should clarify how the final deployment will look like, ideally document it somewhere as well. For example if we run this with gunicorn we should either include a `start.sh`  with the specific command, or extend the `README.md` with instructions. This should also include setting up the database.

**Comment by softbobo** [comment]
The things we need to deploy on `webhooks`:
- [ ] url
- [ ] SSL-Certificate, must be TLS1.0+, can be a supported one or a self-signed one
- [ ] must accept `POST` from subnets `149.154.160.0/20` and `91.108.4.0/22` on port 443, 80, 88, or 8443. 

additionally we need cloud storage and webtraffic to host

**Comment by obitech** [comment]
Do we need webhooks ? Would be good to summarize pro vs. con of webhooks in issue #17

File: bot/start.sh [hunk 1]
```diff
@@ -0,0 +1,25 @@
...
+PORT=8443
+
+# Number of gunicorn Workers
+NUM_WORKERS=2
+
+# Databank directory
+DB_DIR=app/database
+
+# create database
+source $DB_DIR/db_create.py
```
Comment (obitech) [inline bot/start.sh]: Actually I would not include this here and call the setup separately. For example if we call `start.sh` as the primary command to start the bot, and container/process crashes and restart, `db_create.py` would get called again which we don't want.

Comment (softbobo) [reply bot/start.sh]: this is why i used try/except clauses for table creation in `db_create.py`. But might change those clauses to 'pure' SQL like `CREATE TABLE ralph IF NOT EXISTS` ?

Comment (obitech) [reply bot/start.sh]: `CREATE TABLE ralph IF NOT EXISTS` is the cleaner way I think

File: bot/start.sh [hunk 2]
```diff
@@ -0,0 +1,28 @@
...
+# start venv
+source venv/bin/activate
+
+# create database
+source $DB_DIR/db_create.py
+
+# startup server
+exec gunicorn $NAME:app \
+    -b $HOST:$PORT \
+    -w $NUM_WORKERS \
```
Comment (obitech) [inline bot/start.sh]: We also need to pass the API_TOKEN via a variable before the bot is started.

File: bot/start.sh (hunk 1 above)
Comment (softbobo) [reply bot/start.sh]: fixed

File: bot/start.sh (hunk 2 above)
Comment (softbobo) [reply bot/start.sh]: i'd think it's more sensible to do this in the `.env` file and keep the token secret and retrieve it via `load_dotenv()`

**Pull request closed without merging**